deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
hm, that may have been an artifact of having an llvm-config for 3.7 still in usr and not finished running `make cleanall` yet